source/PlanetPanel.cpp

```diff
@@ -230,6 +230,8 @@ void PlanetPanel::TakeOffIfReady()
 	// to take off until the button is clicked again.
 	requestedLaunch = false;
 
+	absentCannotFly.clear();
```

Comment:
wasnt this already cleared by checkwarningsandtakeoff?

Reply:
If the player declines the dialog then no.